modules/020-deckhouse/docs/USAGE.md

```diff
@@ -75,3 +75,17 @@ Manual confirmation of the update to the version `v1.26.0`:
 ```shell
 kubectl patch DeckhouseRelease v1-26-0 --type=merge -p='{"approved": true}'
 ```
+
+### Collect debug info
```

Comment:
It should be a second-level heading I suppose, as it's not linked with the "Setting up the update mode" paragraph.

Reply:
Good point.